osf/models/node.py

```diff
@@ -274,7 +274,12 @@ def get_nodes(self, **kwargs):
         children.
         """
         # Prepend 'child__' to kwargs for filtering
-        filter_kwargs = {'child__{}'.format(key): val for key, val in kwargs.items()}
+        filter_kwargs = {}
+        if 'is_node_link' in kwargs.keys():
```

Comment:
Minor: Calling `.keys()` is unnecessary:

```python
if 'is_node_link' in kwargs:
```

Also, the below two lines can be simplified to:

```python
filter_kwargs['is_node_link'] = kwargs.pop('is_node_link')
```